graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH1.kt

```diff
@@ -0,0 +1,120 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.GetCycle, GraphBehaviours.CreateDirectedGraph, """
+    Check cycles exist in directed graph
```

Comment:
:pick: it is more 'determine' than 'check'
:pick: is it only if we find 'cycle**s**'? one could infer this is 2 or more